gql/cli.py

```diff
@@ -103,6 +103,13 @@ def get_parser(with_examples: bool = False) -> ArgumentParser:
         action="store_true",
         dest="print_schema",
     )
+    parser.add_argument(
+        "--execute-timeout",
+        help="set the execute_timeout argument of the Client (default: 10)",
+        type=int,
+        default=None,
```

Comment:
Shouldn't `default=None` be changed to `default=10`? If not the default would be no timeout. Also, should something like a `-1` be considered as `None`? That would give the user the possibility to set no timeout

Reply:
You're right, I've changed back the default to 10.

To disable the timeout, I propose to allow `--execute-timeout=none` as it is done in my last commit.